modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/DDMFormJSONSerializer.java

```diff
@@ -164,6 +187,22 @@ protected JSONObject toJSONObject(LocalizedValue localizedValue) {
 		return jsonObject;
 	}
 
+	private void _addMissingLocales(
+		LocalizedValue localizedValue, Set<Locale> availableLocales) {
+
+		String defaultLocaleValue = localizedValue.getString(
+			localizedValue.getDefaultLocale());
+
+		Set<Locale> localizedValueAvalilableLocales =
```

Comment:
Hi @pavel-savinov . Can you please fix this typo? Thanks!

Reply:
@jeyvison fixed :)